src/pages/Results/index.js

```diff
@@ -143,14 +146,24 @@ class Results extends Component {
     });
   };
 
+  removeControllerFromFavorites = (event, value, controller) => {
```

Comment:
This looks like almost identical code as found in `Controllers/index.js`, is it?

Reply:
Nice catch!